src/main/java/ch/ivyteam/ivy/maven/InstallEngineMojo.java

```diff
@@ -62,7 +70,26 @@
   public static final String GOAL = "installEngine";
   public static final String ENGINE_LIST_URL_PROPERTY = "ivy.engine.list.url";
   public static final String DEFAULT_ARCH = "Slim_All_x64";
-  
+  public static final String ENGINE_ZIP_GAV_GROUP = "com.axonivy";
+  public static final String ENGINE_ZIP_GAV_ARTIFACT = "engine";
+
+  /**
+   * Indicate if the engine artifact should be downloaded using maven (from a configured maven repository) or if
+   * the URL download way should be used.
+   */
+  @Parameter(property="ivy.engine.download.from.maven", defaultValue = "false")
+  protected Boolean downloadUsingMaven;
```

Comment:
can you cover this flag and its execution behaviour? Other download methods should be widely covered in https://github.com/axonivy/project-build-plugin/blob/master/src/test/java/ch/ivyteam/ivy/maven/TestInstallEngineMojo.java